src/main/java/pl/malek/Calculator/BikeCalculator.java

```diff
@@ -0,0 +1,12 @@
+package pl.malek.Calculator;
+
+import pl.malek.model.Bike;
+
+import java.math.BigDecimal;
+
+public class BikeCalculator implements Calculator{
+    @Override
+    public BigDecimal calculate(Bike bike) {
+        return null;
```

Comment:
jeżeli mamy niezaimplementowane metody to lepiej dać rzucanie wyjątku, potem można się nieźle zdziwić dlaczego coś nie działa ;)